scratchpad/components/todo/bindings/AuthStrategy.ts

```diff
@@ -0,0 +1,17 @@
+import {AUTHENTICATION_STRATEGY} from '@loopback/authentication';
+import {BasicStrategy} from '@passport/basic';
+
+export class AuthStrategy {
+  constructor(
+    private @userRepo() users
+  )
+  name = AUTHENTICATION_STRATEGY; // 'authentication.strategy'
+  value() {
```

Comment:
apologies if this has already been mentioned. This is a general observation. I notice that some of the code has no return type. Its good to use return types in typescript as the caller than knows what it is receiving and it gets typeahead support from most IDE's. For example

```
interface User {
    username: string;
    password: string;
}
``` 
This is just a simple User interface describing the user. Then the value function can be updated

```
value(): User {
...
}
```